.vscode/tasks.json

```diff
@@ -0,0 +1,116 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Run Shell Script",
+      "type": "shell",
+      "command": "sh",
+      "args": ["${file}"],
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "problemMatcher": []
+    },
+    {
+      "label": "Run all python tests",
+      "type": "shell",
+      "command": "./tools/test.sh",
+      "problemMatcher": [],
+      "group": {
+        "kind": "test",
+        "isDefault": true
+      }
+    },
```

Comment:
Just out of curiosity, are these just for documentation purposes?

Reply:
No they're purpose is speed up the workflow. :) 

https://www.loom.com/share/1561aac259c347a9b9d35b335d728830

Reply:
https://github.com/digitalfabrik/integreat-cms/assets/47758554/a2527d39-7007-4ee0-9df2-252f99eef920

:D

Reply:
<img width="436" alt="image" src="https://github.com/digitalfabrik/integreat-cms/assets/13850172/d82d5e88-9de9-4cad-9afc-7ea9b8670bef">

Reply:
<img width="532" alt="image" src="https://github.com/digitalfabrik/integreat-cms/assets/13850172/5feea986-68a3-4fb6-9a0a-cd226081b521">

Reply:
😂 

(Took more time than I care to admit to setup though)